src/pl/senla/hotel/comparators/HotelServicesComparatorByPrice.java

```diff
@@ -1,23 +1,24 @@
 package pl.senla.hotel.comparators;
 
+import pl.senla.hotel.annotations.di.GetInstance;
 import pl.senla.hotel.entity.facilities.Room;
 import pl.senla.hotel.entity.services.Transfer;
 import pl.senla.hotel.entity.services.HotelService;
 import pl.senla.hotel.entity.services.Restaurant;
 import pl.senla.hotel.entity.services.RoomReservation;
 import pl.senla.hotel.repository.Repository;
-import pl.senla.hotel.repository.RepositoryRoomCollection;
 
 import java.util.Comparator;
 
 import static pl.senla.hotel.constant.HotelServiceConstant.ERROR_IN_SERVICE_TYPE;
 
 public class HotelServicesComparatorByPrice implements Comparator<HotelService> {
```

Comment:
to be able to inject fields in it, the class should have AppComponent annotation, or I'm wrong?

Reply:
I'm going to use @AppComponent only for receiving of Classes for Context's creating.
@GetInstance will be used only for receiving of Instances from Context.